src/tests/globalStyles.e2e.ts

```diff
@@ -32,19 +31,58 @@ describe("global styles", () => {
             opacity: opacity
           };
         });
-        expect(noticeAnimation.duration).toEqual("0.3s");
+        expect(noticeAnimation.duration).toEqual("0.15s");
         expect(noticeAnimation.name).toEqual(className.slice(className.indexOf("_") + 2));
         expect(noticeAnimation.opacity).not.toBe("0");
       });
     });
 
-    it("should have initial --calcite-animation-timing CSS Custom Property value", async () => {
-      const page = await newE2EPage({ html: snippet });
-      expect(
-        await page.evaluate(() => {
-          return window.getComputedStyle(document.body).getPropertyValue("--calcite-animation-timing");
-        })
-      ).toBe(" 300ms");
+    it("should set animation duration to 0ms when --animation-timing-factor set to zero", async () => {
```

Comment:
For completeness, can you add a test that asserts on the default transition duration? You already cover disabling animation and not allowing component-level overrides.